src/main/java/data/TaskFileReaderWriter.java

```diff
@@ -0,0 +1,163 @@
+package data;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.Scanner;
+
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.ToDo;
+import utils.Utility;
+
+/**
+ * The TaskFileReaderWriter class provides methods for reading and writing tasks to and from a file.
+ * It includes methods for loading tasks from a file and updating a task file with tasks from a
+ * TaskManager object, and creating a new task file.
+ * @author Nicholas Lee
```

Comment:
I like that you included `@author ` in your comments, something I did not include for my own IP!